test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -7,14 +7,14 @@ namespace Npgsql.Tests;
 
 // This test suite contains ambient transaction tests, except those involving distributed transactions which are only
 // supported on .NET Framework / Windows. Distributed transaction tests are in DistributedTransactionTests.
-[NonParallelizable]
+[NonParallelizable] // All tests are using the exact same table
```

Comment:
Should be possible to convert the tests to use CreateTempTable, can be a separate PR